third_party/conan/docker/utils.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Copyright (c) 2021 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Obtain current docker image tags and digests
+source "$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)/tags.sh"
+source "$(cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd)/digests.sh"
+
+function find_container_for_conan_profile {
+    readonly CONAN_PROFILE="$1"
+
+    # Use a specific tag from the mapping. If none is specified, fall back to `latest`.
+    readonly DOCKER_IMAGE_TAG="${docker_image_tag_mapping[${CONAN_PROFILE}]-latest}"
+
+    CONTAINER="gcr.io/orbitprofiler/${CONAN_PROFILE}:${DOCKER_IMAGE_TAG}"
+
+    if [ ${docker_image_digest_mapping[${CONAN_PROFILE}]+abc} ]; then
```

Comment:
Looks like there is a digest for every profile. Why does this not completely replace the docker_image_tag_mapping and the code above?

Reply:
Can probably be removed now. This mechanism was designed as having fallbacks. But yes, I can probably remove that now. I just thinking whether this has any disadvantages.

Reply:
Fixed!